### ISSUE TITLE
Add support for MarkupContent type

### ISSUE THREAD

**Issue by daviwil** [atom/atom-languageclient#152]
This is a new addition to the v3 protocol, we'll wait for a bit for the API to stabilize before supporting it:

https://github.com/Microsoft/language-server-protocol/commit/692c225dcb737e287e393f148d193877b0a2dd54#diff-7327496955357b9792bab8720dc507d6

**Comment by damieng** [comment]
Does your new PR then address this?

**Comment by daviwil** [comment]
Yep! I think we can close this once that PR is merged.
